examples/webgl_postprocessing_3dlut.html

```diff
@@ -1,207 +1,245 @@
+			document.body.appendChild( container );
 
-						} );
+			camera = new THREE.PerspectiveCamera( 45, window.innerWidth / window.innerHeight, 0.25, 20 );
+			camera.position.set( - 1.8, 0.6, 2.7 );
+
+			scene = new THREE.Scene();
+
+			new RGBELoader()
+				.setPath( 'textures/equirectangular/' )
+				.load( 'royal_esplanade_1k.hdr', function ( texture ) {
+
+					texture.mapping = THREE.EquirectangularReflectionMapping;
+
+					scene.background = texture;
+					scene.environment = texture;
+
+					// model
+
+					const loader = new GLTFLoader().setPath( 'models/gltf/DamagedHelmet/glTF/' );
+					loader.load( 'DamagedHelmet.gltf', function ( gltf ) {
+
+						scene.add( gltf.scene );
 
 					} );
 
-				Object.keys( lutMap ).forEach( name => {
+				} );
+
+			Object.keys( lutMap ).forEach( name => {
 
-					if ( /\.CUBE$/i.test( name ) ) {
+				if ( /\.CUBE$/i.test( name ) ) {
 
-						new LUTCubeLoader()
-							.load( 'luts/' + name, function ( result ) {
+					new LUTCubeLoader()
+						.load( 'luts/' + name, function ( result ) {
 
-								lutMap[ name ] = result;
+							lutMap[ name ] = result;
 
-							} );
+						} );
 
-					} else {
+				} else {
 
-						new LUT3dlLoader()
-							.load( 'luts/' + name, function ( result ) {
+					new LUT3dlLoader()
+						.load( 'luts/' + name, function ( result ) {
 
-								lutMap[ name ] = result;
+							lutMap[ name ] = result;
 
-							} );
+						} );
 
-					}
+				}
 
-				} );
+			} );
 
-				renderer = new THREE.WebGLRenderer();
-				renderer.setPixelRatio( window.devicePixelRatio );
-				renderer.setSize( window.innerWidth, window.innerHeight );
-				renderer.toneMapping = THREE.ACESFilmicToneMapping;
-				renderer.toneMappingExposure = 1;
-				container.appendChild( renderer.domElement );
-
-				const target = new THREE.WebGLRenderTarget( {
-					minFilter: THREE.LinearFilter,
-					magFilter: THREE.LinearFilter,
-					format: THREE.RGBAFormat,
-					colorSpace: THREE.SRGBColorSpace
-				} );
+			renderer = new THREE.WebGLRenderer();
+			renderer.setPixelRatio( window.devicePixelRatio );
+			renderer.setSize( window.innerWidth, window.innerHeight );
+			renderer.toneMapping = THREE.ACESFilmicToneMapping;
+			renderer.toneMappingExposure = 1;
+			container.appendChild( renderer.domElement );
 
-				composer = new EffectComposer( renderer, target );
-				composer.setPixelRatio( window.devicePixelRatio );
-				composer.setSize( window.innerWidth, window.innerHeight );
-				composer.addPass( new RenderPass( scene, camera ) );
-				composer.addPass( new ShaderPass( GammaCorrectionShader ) );
+			composer = new PP.EffectComposer( renderer );
+			composer.setSize( window.innerWidth, window.innerHeight );
+			composer.addPass( new PP.RenderPass( scene, camera ) );
 
-				lutPass = new LUTPass();
-				composer.addPass( lutPass );
+			const gammaShaderMaterial = new THREE.ShaderMaterial( GammaCorrectionShader );
+			composer.addPass( new PP.ShaderPass( gammaShaderMaterial, 'tDiffuse' ) );
```

Comment:
See above.

Reply:
Removing the Gamma Correction pass here results in a different visual:
See the comparison below : 

# With gamma correction pass
![With gamma correction pass](https://github.com/mrdoob/three.js/assets/18304870/abe34b2d-4e24-402d-9a0b-25dd09ca6bd1)

# Without gamma correction pass
![Without gamma correction pass](https://github.com/mrdoob/three.js/assets/18304870/48619642-6ff3-43ec-8bc9-f2c99dfa640a)

# Original example from threejs.org
![Original example from threejs.org](https://github.com/mrdoob/three.js/assets/18304870/d985de65-1389-4aa2-801d-71d609cdbb58)

The fact that the LUT pass is fed with the gamma correction pass output is meaningful for the final result. The outputColorSpace occurs at the end of the pipeline. For this example, I think we should keep the intermediary gamma correction pass.

Reply:
Ah good call – yes, if the LUT requires sRGB input then we should use OutputPass before the LUT. Possibly there are other ways to handle this in postprocessing but we don't need to consider that now. Thanks!